src/components/dropdown.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference path="../../typings/tsd.d.ts" />
+import {Component, View, NgClass} from 'angular2/angular2';
+
+@Component({
+  selector: 'uib-dropdown',
+  properties: [
+    'open'
+  ]
+})
+@View({
+  directives: [NgClass],
+  template: `<div class="dropdown" [ng-class]="{open: open}">
```

Comment:
Pawel had a suggestion in the Angular repo when I mentioned not being able to use directives - I need to try
```js
properties: [
  '[class.open]: open'
]
```